test/quantization/test_mixed_precision.py

```diff
@@ -0,0 +1,46 @@
+import torch
+import torch.nn as nn
+
+import os
+import sys
+# append the path to the naive_intNwo.py file
+sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))), "torchao/quantization/prototype/mixed_precision/scripts"))
```

Comment:
Does it work if you just add an empty `__init__.py` to `torchao/quantization/prototype/mixed_precision`? Then you won't need this line anymore?

Reply:
Added __init__.py and removed the path append